docs/bare_metal/mbed2_porting.md

```diff
@@ -19,7 +19,7 @@ The configure your target:
 1. Run the following command:
```

Comment:
Should we change the title on line 17, too?

Reply:
@iriark01 Done. Please re-review